src/Database/Pgsql/license.md

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
This is an old license, please replace with the new one

Reply:
*groan* stop changing things.... haha

Yeah, I'll get that updated